core/internal/featurechecker/features.go

```diff
@@ -0,0 +1,115 @@
+package featurechecker
+
+import (
+	"context"
+	"sync"
+
+	"github.com/Khan/genqlient/graphql"
+	"github.com/wandb/wandb/core/internal/gql"
+	"github.com/wandb/wandb/core/internal/observability"
+	spb "github.com/wandb/wandb/core/pkg/service_go_proto"
+)
+
```

Comment:
how about adding this:
```suggestion
type ServerFeatures map[spb.ServerFeature]ServerFeature
```